src/main/java/gov/usgs/owi/nldi/controllers/GlobalDefaultExceptionHandler.java

```diff
@@ -38,8 +39,12 @@ public static String handleError(Exception ex, HttpServletResponse response) {
 				//This exception's message contains implementation details after the new line, so only take up to that.
 				return ex.getLocalizedMessage().substring(0, ex.getLocalizedMessage().indexOf("\n"));
 			} else {
-				return ex.getLocalizedMessage().replaceAll("([a-zA-Z]+\\.)+","");
+				return ex.getLocalizedMessage().replaceAll("([a-zA-Z]+\\.)+", "");
 			}
+		} else if (ex instanceof ResponseStatusException) {
+			ResponseStatusException rse = (ResponseStatusException)ex;
+			response.setStatus(rse.getStatus().value());
+			return rse.getLocalizedMessage();
```

Comment:
Following the pattern of the other exceptions, only the status (404) is returned in the response.  It might be nice to include more info in the response, such as 'The basin could not be found'.  A user would assume that, but it would allow them to be confident of what is happening.

Reply:
This is class only meant for unhandled exceptions. An expected (reasonable) not found condition should be handled in the code where it occurs.